README.adoc

```diff
@@ -154,54 +157,35 @@ Wait for the `Press the Enter key to run tests on demand.` message, and then pre
  Integration tests finished.
 ----
 
-To begin bootstrapping, annotate the `src/test/java/io/openliberty/guides/testing/PersonServiceIT.java` class with the `@MicroShedTest` annotation. This annotation indicates that the test class uses MicroShed Testing.
+To begin bootstrapping, import the [hotspot=importMicroShedTest file=0]`MicroShedTest` annotation and annotate the `PersonServiceIT` class with [hotspot=microShedTest file=0]`@MicroShedTest`. This annotation indicates that the test class uses MicroShed Testing. 
 
-[role="code_command hotspot file=0", subs="quotes"]
-----
-#Update the `PersonServiceIT` class.#
-`src/test/java/io/openliberty/guides/testing/PersonServiceIT.java`
-----
-[role="edit_command_text"]
-Import the [hotspot=importMicroShedTest file=0]`MicroShedTest` annotation and annotate the `PersonServiceIT` class with [hotspot=microShedTest file=0]`@MicroShedTest`.
-
-PersonServiceIT.java
-[source, java, linenums, role='code_column hide_tags=copyright']
-----
-include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.1.java[]
-----
-// if use PersonServiceIT.2.java, use following:
-//[source, java, linenums, role='code_column hide_tags=copyright,importInject,importMPApp,importAssertNotNull,importSharedContainerConfig,sharedContainerConfig,inject,personSvc,mpApp,container,testCreatePerson']
-//----
-//include::hotspots/src/test/java/io/openliberty/guides/testing/PersonServiceIT.2.java[]
-//----
-
-Next, the `PersonServiceIT` class outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
+The `PersonServiceIT` class outlines some basic information that informs how MicroShed Testing starts the application runtime and at which URL path the application will be available:
```

Comment:
Change "at which URL path the application will be available:" to "at which URL path the application is available".